src/earn/poolInfoScreen/BeforeDepositBottomSheet.tsx

```diff
@@ -226,48 +296,73 @@ export default function BeforeDepositBottomSheet({
     depositTokenId: pool.dataProps.depositTokenId,
   }
 
+  const showCrossChainSwap = canSwapDeposit && hasTokensOnOtherNetworks
+  const showSwap = !canSwapDeposit && (hasTokensOnSameNetwork || hasTokensOnOtherNetworks)
+  const showAdd = canAdd && !hasDepositToken
+  const showAddMore =
+    canAdd && hasDepositToken && !(hasTokensOnSameNetwork && hasTokensOnOtherNetworks)
```

Comment:
This is for [this case](https://www.figma.com/design/E1rC3MG74qEg5V4tvbeUnU/Earn?node-id=8300-99281&node-type=instance&t=041j9AtxqFboP657-0), where add isn't shown if a user can deposit and also has same AND differnet chain tokens.